src/main/java/uk/gov/hmcts/sptribs/caseworker/service/HearingService.java

```diff
@@ -50,7 +50,8 @@ public void addListingIfExists(CaseData data) {
                 firstListing.getValue().setHearingCancellationReason(data.getRetiredFields().getCicCaseHearingCancellationReason());
                 firstListing.getValue().setCancelHearingAdditionalDetail(data.getRetiredFields().getCicCaseCancelHearingAdditionalDetail());
                 firstListing.getValue().setPostponeReason(data.getRetiredFields().getCicCasePostponeReason());
-                firstListing.getValue().setPostponeAdditionalInformation(data.getRetiredFields().getCicCasePostponeAdditionalInformation());
+                firstListing.getValue().setPostponeAdditionalInformation(data.getRetiredFields()
```

Comment:
@sjamihmcts - can you revert back if there is no actual change here?

Reply:
Done, reverted